src/store/selectors/activities/types.ts

```diff
@@ -94,26 +99,22 @@ export type ActivityContent = {
    * display name in the format of a complete sentence.
    */
   description?: (string | CurrencyValue)[];
-} & (
-  | {}
-  // This union type enforces that if a body is present, there must be a next
-  // activity in the chain. If this turns out to be too restrictive, we can
-  // loosen the requirement.
-  | {
-      /**
-       * A larger, detailed body describing (in text) or representing (visually) the
-       * action.
-       */
-      body: ActivityBody;
-      /**
-       * If this is a chained activity, the next piece of content that has occurred
-       * in the chain.
-       */
-      nextInChain: {
-        direction: ActivityDirection;
-        content: ActivityContent;
-      };
-    });
+  /**
+   * A larger, detailed body describing (in text) or representing (visually) the
+   * action.
+   *
+   * Currently, a body is required if there is to be further chained content. If
+   * that is too restrictive, we can relax that restriction, but it will likely
+   * require a visual redesign as well.
+   */
+  body?: {
```

Comment:
This causes a greater deal of nesting in the data structure, but more naturally represents that chained content comes after body content, always. if that's no longer the case at some point then we can split this up again, but the visual design wouldn't make that look particularly clean, so that would require some rethinking.